### ISSUE TITLE
Issues with Keyboard drivers on QEMU

### ISSUE THREAD

**Issue by Pineconium** [steelsofliquid/NanamiOS#1]
I tried using your OS on QEMU, and while the OS did load and the pseudo-mouse drivers work, the keyboard drivers didn't. But it worked fine on VirtualBox.

**Comment by steelsofliquid** [comment]
So, I know it's been over a year and the NanamiOS repository in particular hasn't been updated outside of housekeeping changes in quite a long time, but after having tested NanamiOS' successor (OpenSteel/OS) in QEMU numerous times, I've found that for some odd reason, in QEMU, the mouse driver will break the keyboard. When the mouse driver is disabled, the keyboard works perfectly fine.

I found this when I was rewriting the driver model and testing the mouse driver in VirtualBox, which continues to exhibit the same sort of behaviour.

TL;DR - I've done some testing on OpenSteel/OS and was able to replicate the issue (OpenSteel/OS being built off of the NanamiOS codebase, the mouse drivers are nearly identical).
